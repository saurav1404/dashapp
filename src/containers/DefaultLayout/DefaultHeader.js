import React, { Component } from 'react';
import Navigation from '../../Components/Navigation/Navigation'
import PropTypes from 'prop-types';

const propTypes = {
  children: PropTypes.node,
};

const defaultProps = {};

class DefaultHeader extends Component {
  render() {
    return (
      <Navigation { ...this.props } />
    );
  }
}
DefaultHeader.propTypes = propTypes;
DefaultHeader.defaultProps = defaultProps;
export default DefaultHeader;
