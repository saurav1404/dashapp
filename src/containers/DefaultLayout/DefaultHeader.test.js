import React from 'react';
import DefaultHeader from './DefaultHeader';
import { shallowToJson } from 'enzyme-to-json';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() })

describe('DefaultHeader test cases', () => {
  let output;
  beforeEach(() => {
    output = shallow(
      <DefaultHeader />
    );
  });

  afterEach(() => {
    output = null;
  });

  it('should render correctly', () => {
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});