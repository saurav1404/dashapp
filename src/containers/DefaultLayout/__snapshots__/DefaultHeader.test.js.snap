// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultHeader test cases should render correctly 1`] = `<Connect(Navigation) />`;
